app.R

```diff
@@ -6,24 +6,23 @@ library(ggplot2)
 # Reload when saving the app
 options(shiny.autoreload = TRUE)
 
+# Load race results data
 race_results <- readr::read_csv("data/formula1_2021season_raceResults.csv")
 race_results$Track <- factor(race_results$Track, levels = unique(race_results$Track))
 race_results <- race_results |>
   dplyr::group_by(Driver) |>
   dplyr::mutate(cumpoints = cumsum(Points))
 
+# Load race names
+race_table <- readr::read_csv("data/formula1_2021season_raceResults.csv")
```

Comment:
This part is a bit redundant. We can just use `race_results` to get `race_table` without needing to reread in the entire dataset and changing the Track column to factor since that was already done above.

Reply:
Good idea, edited